app/models/user.rb

```diff
@@ -29,4 +30,45 @@ def remember
   def forget
     update_attribute(:remember_digest, nil)
   end
+
+  # Activates an account.
+  def activate
```

Comment:
```suggestion
  def activate_account
```

And kill the comment